2-mandatory/3-shopping-list.js

```diff
@@ -20,6 +20,14 @@ let pantry = {
 
 function createShoppingList(recipe) {
   // write code here
+  let result = {};
+  result.name = recipe["name"];
+  let pantryArr = pantry.fridgeContents.concat(pantry.cupboardContents);
```

Comment:
I really like how you use `concat` here to combine the fridge & cupboard items first to make it easier to search through